data/json/recipes/other/medical.json

```diff
@@ -111,5 +111,35 @@
       [ [ "cooking_oil", 2 ] ],
       [ [ "thyme", 2 ] ]
     ]
-  }
+  },
+  {
+    "type": "recipe",
+    "result": "disinrag",
+    "category": "CC_OTHER",
+    "subcategory": "CSC_OTHER_MEDICAL",
+    "skill_used": "firstaid",
+    "skills_required": [ "firstaid", 1 ],
```

Comment:
Skills required and skill used should not overlap. Use one or the other, or make one of them a different skill.